src/Ninja/G1CP/Content/Fixes/Session/fix278_EN_WomanOuMismatch.d

```diff
@@ -0,0 +1,7 @@
+/*
+ * #278 Ambient women dialog subtitles don't match the audio (EN)
+ */
+func int G1CP_278_EN_WomanOuMismatch() {
```

Comment:
The name of the function should also reflect that it is an SVM, e.g. `G1CP_278_EN_WomanSvmOuMismatch`.